examples/chart/teleport-kube-agent/templates/_helpers.tpl

```diff
@@ -0,0 +1,17 @@
+{{- define "teleport.kube.agent.isUpgrade" -}}
+{{- /* Checks if action is an upgrade from an old release that didn't supported Secret storage */}}
+{{- /* TODO: change the behaviour for Teleport 11 */}}
```

Comment:
Did you mean for Teleport 12 here? Cause this is going to Teleport 11 - does anything need to change here still?

Reply:
This was a comment I added for T10.1 but forgot to remove it yesterday.